tests/utils/test_gcn.py

```diff
@@ -0,0 +1,11 @@
+import mlx.core as mx
+import pytest
+
+from mlx_graphs.nn.conv.gcn_conv import GCNConv
+
+
+def test_inv_degree():
+    deg = mx.array([1, 2, 3, 0])
+    expected = mx.array([1, 0.707107, 0.57735, 0])
+    out = GCNConv._deg_inv_sqrt(deg)
+    assert mx.array_equal(out, expected), "Incorrect degree inversion"
```

Comment:
Could you add a test with an inf float?

```python
mx.array([1, 2, 3, float("inf")])
```

Reply:
What i mean is that mlx right now has issues dealing with inf values when using mx.where so an example where there is a 0 in the list before the **-0.5 actually corresponds to an inf if we hadn't add an epsilon but i agree it's too ambiguous, will add documentation and rechange names to make it more explicit